boards/arm/circuitdojo_feather_nrf9160/circuitdojo_feather_nrf9160_common.dts

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright (c) 2018-2020 Nordic Semiconductor ASA
+ * Copyright (c) 2020 Circuit Dojo LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	model = "Circuit Dojo nRF9160 Feather";
+	compatible = "circuitdojo,feather-nrf9160";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		blue_led: led_0 {
+			gpios = <&gpio0 3 GPIO_ACTIVE_LOW>;
+			label = "Green LED (D7)";
```

Comment:
Is it a blue LED or a green one?

Reply:
@jaredwolff ^^

Reply:
It's blue. I'll rename the label to be consistent.